torch/csrc/shim_common.cpp

```diff
@@ -225,6 +230,65 @@ AOTI_TORCH_EXPORT AOTITorchError torch_library_impl(
   });
 }
 
+AOTI_TORCH_EXPORT AOTITorchError torch_create_device(
+    int32_t device_type,
+    int32_t device_index,
+    DeviceHandle* ret_device) {
+  AOTI_TORCH_CONVERT_EXCEPTION_TO_ERROR_CODE({
```

Comment:
We should eventually wean off the AOTI macros too...not within scope for this PR, but just flagging as another thing we should do